zib/StructureDefinitions/zib-BarthelADLIndex.xml

```diff
@@ -0,0 +1,412 @@
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The Barthel ADL index is a validated scale used to measure and track patients’ activities of daily living (ADL) and indirectly, the extent to which they are dependent on help.&#xD;&#xA;The list comprises 10 variables: bowels, bladder, grooming, toilet use, feeding, transfers (from chair to bed and vice versa), mobility, dressing, climbing stairs and bathing, which are all given a score.&#xD;&#xA;The total score is the sum of the scores of the 10 variables." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) BarthelADLIndex v3.2 (2024)](https://zibs.nl/wiki/BarthelADLIndex-v3.2(2024EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bartheladlindex-v3.2-2024EN" />
+    <uri value="https://zibs.nl/wiki/BarthelADLIndex-v3.2(2024EN)" />
+    <name value="zib BarthelADLIndex-v3.2(2024EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BarthelADLIndex" />
+      <alias value="BarthelIndex" />
+      <mapping>
+        <identity value="zib-bartheladlindex-v3.2-2024EN" />
+        <map value="NL-CM:4.2.1" />
+        <comment value="BarthelADLIndex" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="273302005" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueInteger" />
+      <short value="TotalScore" />
+      <definition value="The Barthel ADL index is the sum of the scores of the individual variables. The interpretation of this total score is 0-9 for serious dependence, 10-19 for slight dependence and 20 for independence. Twenty is the maximum score." />
```

Comment:
The definition text contains a newline, which is missing here. Do you know why?

Reply:
Fixed